engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -60,27 +61,17 @@
     private static final int STACKS = 128;
     private static final int RADIUS = 1024;
 
-    @In
-    private Config config;
-
-    @In
-    private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
-
-    @In
     private WorldRenderer worldRenderer;
 
-
-    private Camera playerCamera;
     private int skySphere = -1;
     private SetWireframe wireframeStateChange;
 
-    /**
-     * This method must be called once shortly after instantiation to fully initialize the node
-     * and make it ready for rendering.
-     */
-    @Override
-    public void initialise() {
-        playerCamera = worldRenderer.getActiveCamera();
+    public BackdropNode(Context context) {
+        Config config = context.get(Config.class);
+        DisplayResolutionDependentFBOs displayResolutionDependentFBOs = context.get(DisplayResolutionDependentFBOs.class);
+        worldRenderer = context.get(WorldRenderer.class);
+
+        Camera playerCamera = worldRenderer.getActiveCamera();
```

Comment:
Comments above apply to this node too: please check the need to store the Config and the WorldRenderer.